packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -640,6 +641,7 @@ export const LOCUS = {
 export const LOCUSINFO = {
   EVENTS: {
     CONTROLS_RECORDING_UPDATED: 'CONTROLS_RECORDING_UPDATED',
+    CONTROLS_MEETING_CONTAINER_UPDATED: 'CONTROLS_MEETING_CONTAINER_UPDATED',
```

Comment:
To confirm, is this a new locus-info update?

Reply:
Yes, the meetingContainerUrl is in locusInfo.controls.meetingContainer.  I tried to follow the example of locusInfo.controls.record, even though this is a little more simple than that.